store/src/java/com/zimbra/cs/service/FileUploadServlet.java

```diff
@@ -719,33 +804,10 @@ public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOEx
         Upload up = new Upload(acct.getId(), fi, filename);
 
         if (filename.endsWith(".har")) {
```

Comment:
Why do we need separate for .har? 
I think we have discuss to handle everything at one place.

Reply:
.har should be added as default config.

Reply:
Removed the unnecessary code for handling the .har file as it was based upon only the extension. As, if we remove the extension of the .har file ZBUG-711 still exist and making the web-client unresponsive and also in CPU spike. While it is correctly handled by the tika with it's content-type detected as application/pdf contrary to earlier where it is getting detected as application/octet-stream but the file actually contains html content and the convertd is trying to convert the document based on application/octet-stream and failing.